src/java.base/share/classes/sun/security/util/math/intpoly/MontgomeryIntegerPolynomialP256.java

```diff
@@ -103,8 +104,8 @@ public ImmutableElement getElement(BigInteger v) {
         setLimbsValuePositive(v, vLimbs);
 
         // Convert to Montgomery domain
-        int numAdds = mult(vLimbs, h, montLimbs);
-        return new ImmutableElement(montLimbs, numAdds);
+        mult(vLimbs, h, montLimbs);
+        return new ImmutableElement(montLimbs, 0);
```

Comment:
The rest of the changes in 30e0ca9ad7122b50a7db2530c9cfa3c4a57b616a are missing here.

Reply:
Use the upstream version, also `src/java.base/share/classes/sun/security/util/math/intpoly/IntegerPolynomial.java`, `src/java.base/share/classes/sun/security/util/math/intpoly/IntegerPolynomial1305.java`, `src/java.base/share/classes/sun/security/util/math/intpoly/IntegerPolynomialModBinP.java` and `make/jdk/src/classes/build/tools/intpoly/FieldGen.java`.